olm-catalog/serverless-operator/project.yaml

```diff
@@ -22,9 +22,12 @@ requirements:
 
 dependencies:
   serving: 0.21.0
+  kourier: 0.21.0
+  net_istio: 0.21.0
+  maistra: 2.0.0
+
   eventing: 0.21.0
   eventing_kafka: 0.20.0
-  kourier: 0.21.0
+
   cli: 0.20.0
-  maistra: 2.0.0
```

Comment:
Why was maistra already a dependency before (re-)adding net_istio again ?

Reply:
Because we use the productized envoy that maistra uses for our Kourier deployment, so we don't have to be in the business of compiling (and patching) envoy.